### ISSUE TITLE
[Feature] - Add option to skip TokensManager in fights when untaxed to save gas

### ISSUE THREAD

**Issue by poshdan** [CryptoBlades/cryptoblades#1771]
### Prerequisites

- [ ] I checked to make sure that this feature has not already been filed
- [ ] I'm reporting this information to the correct repository
- [X] I understand enough about this issue to complete a comprehensive document

### Describe the feature and its requirements

Contract jumps have a gas cost that is important to consider on expensive networks.
I estimate we would save 6k gas by skipping the TokensManager (whose role is to take extra gas on fights), which is about 0.00003 BNB, could be ~one cent saved.

Since on BSC we are not taking the tax during the event, we want to save the users that gas. We do want to keep the feature working on alternate networks though, so this has to be a configuration option in the CryptoBlades file.

- Create new function in cryptoblades for "untaxed fight", make the existing fight function into an override of a hidden "_fight" function that the untaxed fight will also call
- Create new var (NOT STATE VARIABLE, a VAR CONSTANT) in cryptoblades that is used in a require statement in the untaxed fight function compared with "1" (for being on)
- Alter frontend to read the value of that var (cryptoblades.vars(key)), and call the untaxed function in cryptoblades instead of the tokensmanager fight function if the var is set to 1.

### Is your feature request related to an existing issue? Please describe.

part of doc #1761

### Is there anything stopping this feature being completed?

Fight ABI will change with Equipment manager (won't take a weapon id parameter, and the internal code changes a LOT) so this should be worked on once that is merged into main

### Describe alternatives you've considered

0

### Additional context

Contact PoshDan#7875
